src/jackdaw/streams/interop.clj

```diff
@@ -396,6 +408,14 @@
                  ^KTable (ktable* other-ktable)
                  ^ValueJoiner (value-joiner value-joiner-fn))))
 
+  (suppress
+    [_ {:keys [max-records max-bytes until-time-limit-ms]
```

Comment:
Seems like you no longer need to destructure the `suppress` argument here.

Reply:
Yes, updated the code and also removed the `let` binding. No need for it